app/packs/src/apps/mydb/elements/details/researchPlans/researchPlanTab/ResearchPlanDetailsFieldImage.js

```diff
@@ -109,6 +148,36 @@ export default class ResearchPlanDetailsFieldImage extends Component {
     );
   }
 
+  renderImageEditedWarning(attachment){
+    if(!attachment) {return null;}
+    if(!attachment.updatedAnnotation){return null;} 
+    
+    return (
+      <div className='imageEditedWarning'>
+        <Alert>Image was edited. Please save Researchplan to apply changes</Alert>
+      </div>
```

Comment:
Use `app/packs/src/apps/mydb/elements/details/researchPlans/SaveResearchPlanWarning.js`?

Reply:
The hole function was replaced by the SaveResearchPlanWarning component and is now unused, I removed it.